renovate-config.js

```diff
@@ -12,21 +14,37 @@ module.exports = {
             "groupSlug": "all-patch",
             "matchPackageNames": ["*"],
             "matchUpdateTypes": ["patch"],
+            "postUpgradeTasks": {
+                "commands": ["npm install --package-lock-only"],
+                "fileFilters": ["**/**"],
+                "executionMode": "branch"
+            }
         },
         {
             //for v2.x.x make dashboard approval to all major and minor dependencies updates
             "matchBaseBranches": ["v2.x.x"],
             "matchUpdateTypes": ["major", "minor"],
             "dependencyDashboardApproval": true,
         },
+        // Disable the upgrade of springboot in v3.x.x
+        {
+            "matchBaseBranches": ["v3.x.x"],
+            "matchPackageNames": ["org.springframework.boot:**"],
+            "enabled": false
+        },
```

Comment:
Does this disable the automatic update of spring boot? why?

Reply:
I guess this isn't needed given the fix Andrej prepared

Reply:
Spring boot upgrade to v3.3.5 breaks the gateway. There is another ticket opened to fix it. Once the fix is integrated we can remove this code.

Reply:
right, but the fix is in https://github.com/zowe/api-layer/pull/3882, I think we don't need to ad it to be rolled back immediately after.

Reply:
oh seems like #3882 got approved 10 min ago. When I opened this PR there was no fix for spring boot yet in v3.x.x. I will remove this code then.